core/object/make_virtuals.py

```diff
@@ -198,6 +200,37 @@ def run(target, source, env):
 #define GDVIRTUAL_TRACK(m_virtual, m_initialized)
 #endif
 
+// MSVC WORKAROUND START
+// FIXME The below helper functions are needed to work around an MSVC bug.
+// They should be removed once the bug ceases to be triggered.
```

Comment:
Maybe add a not here also that this requires a change to `core/object/make_virtuals.py` when this can be removed